src/client/pages/Payment/Payment.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { fonts, colorsV3 } from '@hedviginsurance/brand'
+import { SessionTokenGuard } from 'containers/SessionTokenGuard'
+import { HedvigLogo } from 'components/icons/HedvigLogo'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useTextKeys } from 'utils/textKeys'
+import { useCreateTrustlyURL } from './useCreateTrustlyURL'
+import { TrustlyIframe } from './TrustlyIframe'
+import { useStoredRedirectURL } from './useStoredRedirectURL'
+
+export const PaymentPage = () => {
+  const textKeys = useTextKeys()
+  const { path: pathLocale } = useCurrentLocale()
+  const redirectURL = useStoredRedirectURL()
+  const [, trustlyURL] = useCreateTrustlyURL()
+
+  const handleSuccess = (baseRedirectURL: string) => {
```

Comment:
Nit: if you capture redirectUrl here, you can simplify usage.  It would also enable adding `useCallback` when needed
```
onSuccess={handleSuccess}
```